### ISSUE TITLE
ICSharpCode.SharpZipLib

### ISSUE THREAD

**Issue by tobygose** [polterguy/phosphorusfive#10]
I just downloaded latest version and started to play with it. Once I got to the point of running webapp I got errors looking for ICsharpCode with version below. 
Can you tell me where you got that version? I only find 0.4.... ones on web.

<add assembly="ICSharpCode.SharpZipLib, Version=4.84.0.0, Culture=neutral, PublicKeyToken=1b03e6acf1164f73" />

Thanks
Toby

**Comment by polterguy** [comment]
Hi Toby, for some reasons this one eluded my email inbox ...? :P
Sorry for late answer, which version are we talking about. The latest download, the latest *stable* download, or the code through Git ...?

**Comment by tobygose** [comment]
I am sorry I have not gotten back with you. I just downloaded the latest code through Get and was trying to run the webapp when I get the error. I commented out the code in the web.config and it works. Below is the code I commented out.

**Comment by polterguy** [comment]
Thx Toby, that seems to be a bug in the web.config yes, since it's not supposed to reference that assembly. The CSharpZip library, was used as a plugin assembly at some point, to create support for zipping and unzipping, with AES cryptography, files and folders. But I took it out temporary, to make sure I could wrap up a stable release ...

Thx for notifying me, this will be fixed in the next release :)

**Comment by tobygose** [comment]
Thank you for the quick reply, I have enjoyed playing with this and learning it. I find it very interesting.  Can you give me link to the latest download with out having to use the code? I would like to get this on web server and see how far I can take it. 

Thanks again

**Comment by polterguy** [comment]
Of course Toby,

I'd encourage you to use the latest *stable* release, which is the second last release at the time of this writing, in [this list](https://github.com/polterguy/phosphorusfive/releases).

Notice the release at the top of that list, is a BETA release of the next release, and I'd in general not encourage you to use that one at the moment, but rather the one that is clearly marked as **Latest release**.

I assume you've seen [this](https://github.com/polterguy/phosphorusfive-dox) ...?
The above is a *"jump start guide"*, taking you through most of the *core stuff*, and my reasoning for its architectural decisions, etc, and should be possible to skim through in about a day or two ...

Have fun :)

**Comment by tobygose** [comment]
Yes I have been spending a lot of time on your guide. But I run all in VS right now. I just would like to get solid build on web server. 

Thanks again for your help and time

**Comment by polterguy** [comment]
Great, let me know if you need any assistance :)
